patches/react-beautiful-dnd+13.1.1.patch

```diff
@@ -0,0 +1,259 @@
+diff --git a/node_modules/react-beautiful-dnd/dist/react-beautiful-dnd.cjs.js b/node_modules/react-beautiful-dnd/dist/react-beautiful-dnd.cjs.js
+index 12884e9..3a16e5e 100644
+--- a/node_modules/react-beautiful-dnd/dist/react-beautiful-dnd.cjs.js
++++ b/node_modules/react-beautiful-dnd/dist/react-beautiful-dnd.cjs.js
+@@ -6227,8 +6227,10 @@ function useTouchSensor(api) {
+           x: clientX,
+           y: clientY
+         };
++        var handle = api.findClosestDragHandle(event);
++        !handle ? process.env.NODE_ENV !== "production" ? invariant(false, 'Touch sensor unable to find drag handle') : invariant(false) : void 0;
+         unbindEventsRef.current();
+-        startPendingDrag(actions, point);
++        startPendingDrag(actions, point, handle);
+       }
+     };
+   }, [api]);
+@@ -6268,7 +6270,7 @@ function useTouchSensor(api) {
+       phase.actions.abort();
+     }
+   }, [stop]);
+-  var bindCapturingEvents = useMemoOne.useCallback(function bindCapturingEvents() {
++  var bindCapturingEvents = useMemoOne.useCallback(function bindCapturingEvents(target) {
```

Comment:
Hi @kosmydel . I am working on ["Create a fork for react-beautiful-dnd and remove patch"](https://github.com/Expensify/App/issues/28797) and I need to define a type for ```target``` param. So what should it be? https://github.com/Expensify/react-beautiful-dnd/pull/1#discussion_r1369492676

Reply:
Hey @DylanDylann, thanks for the question!

I think it will be `HTMLElement`. 

[Here is the original PR, with this change.](https://github.com/atlassian/react-beautiful-dnd/pull/2281/files#diff-49722d6e0b2a24d20cb31abbf95e34dc0e2a958f1efe226492265ed0a9a02340) And it is already typed.

Reply:
Many thanks @kosmydel